### PR TITLE
cockpit-*: add page

### PR DESCRIPTION
**Pull request by ohyahgit**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
> Seems like I changed the line ending, Will update it soon.

File: pages/linux/cockpit-bridge.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# cockpit-bridge
+
+> Used to relay messages and commands between the frontend and server in the cockpit suite.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-bridge.1.html>.
+
+- List all cockpit packages:
+`cockpit-bridge --packages`
+
+- Show command help:
```
Comment (kbdharun) [inline pages/linux/cockpit-bridge.md:9]:
```suggestion
- Display help:

```

File: pages/linux/cockpit-bridge.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# cockpit-bridge
+
+> Used to relay messages and commands between the frontend and server in the cockpit suite.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-bridge.1.html>.
+
+- List all cockpit packages:
```
Comment (kbdharun) [inline pages/linux/cockpit-bridge.md:6]:
```suggestion
- List all cockpit packages:

```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, Thanks for your contribution. 

Some tips:
- Refer to our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) in future.
- Don't move the links to the next line.
- End page description with a period.
-  Add an empty line between a command and its description.

I did some changes for you already.

File: pages/linux/cockpit-ws.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# cockpit-ws
+
+> The web server component of the cockpit web-based server management interface.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-ws.8.html>.
+
+- Set custom ip address and port:
+`cockpit-ws --address {{10.10.0.154}} --port {{8080}}`
+
+- Show command help:
```
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:9]:
```suggestion
- Display help:

```

**Review by kbdharun — COMMENTED** [review]
These changes will fix the linter errors, I will review the content later today.

File: pages/linux/cockpit-ws.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# cockpit-ws
+
+> The web server component of the cockpit web-based server management interface.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-ws.8.html>.
+
+- Set custom ip address and port:
```
Comment (kbdharun) [inline pages/linux/cockpit-ws.md:6]:
```suggestion
- Set custom ip address and port:

```

File: pages/linux/cockpit-tls.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# cockpit-tls
+
+> A HTTP proxy that is used to encrypt traffic between a client and cockpit-ws.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-tls.8.html>.
+
+- Set port:
+
+`cockpit-tls --port {{443}}`
+
+- Show command help:
```
Comment (kbdharun) [inline pages/linux/cockpit-tls.md:10]:
```suggestion
- Display help:
```

**Comment by kbdharun** [comment]
Tip: Batch all suggestions in `Files changed` tab and make a single commit.

File: pages/linux/cockpit-bridge.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# cockpit-bridge
+
+> Used to relay messages and commands between the frontend and server in the cockpit suite.
```
Comment (kbdharun) [inline pages/linux/cockpit-bridge.md:3]:
```suggestion
> Relay messages and commands between the front end and server in the cockpit suite.
```

File: pages/linux/cockpit-desktop.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+# cockpit-desktop
+
+> Provides secure access to Cockpit pages in am already running session.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-desktop.1.html>.
+
```
Comment (kbdharun) [inline pages/linux/cockpit-desktop.md:5]:
```suggestion

- Open a page:

`cockpit-desktop {{URL}} {{SSH_Host}}`
```
I think we can add the general command while keeping the example below too.

File: pages/linux/cockpit-tls.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# cockpit-tls
+
+> A HTTP proxy that is used to encrypt traffic between a client and cockpit-ws.
```
Comment (kbdharun) [inline pages/linux/cockpit-tls.md:3]:
```suggestion
>  TLS terminating HTTP proxy to encrypt traffic between a client and `cockpit-ws`.
```

File: pages/linux/cockpit-tls.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://cockpit-project.org/guide/latest/cockpit-tls.8.html>.
+
+- Set port:
+
+`cockpit-tls --port {{443}}`
```
Comment (kbdharun) [inline pages/linux/cockpit-tls.md:8]:
```suggestion
`cockpit-tls --port {{PORT}}`
```
I guess we can have this, in order to better align with the man page.

File: pages/linux/cockpit-tls.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+# cockpit-tls
+
+> A HTTP proxy that is used to encrypt traffic between a client and cockpit-ws.
+> More information: <https://cockpit-project.org/guide/latest/cockpit-tls.8.html>.
+
+- Set port:
```
Comment (kbdharun) [inline pages/linux/cockpit-tls.md:6]:
```suggestion
- Serve HTTP requests to a specific PORT instead of port `9090`:
```

**Review by kbdharun — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**